pymeasure/instruments/siglenttechnologies/siglent_sds1072cml.py

```diff
@@ -0,0 +1,309 @@
+        (verticalGain,)=struct.unpack_from('f',descriptor,offset=descriptorOffset+156)
+        (verticalOffset,)=struct.unpack_from('f',descriptor,offset=descriptorOffset+160)
+        (horizInterval,)=struct.unpack_from('f',descriptor,offset=descriptorOffset+176)
+        (horizOffset,)=struct.unpack_from('d',descriptor,offset=descriptorOffset+180)
+        descriptorDictionnary={
+            "numDataPoints":numDataPoints,
+            "verticalGain":verticalGain,
+            "verticalOffset":verticalOffset,
+            "horizInterval":horizInterval,
+            "horizOffset":horizOffset,
+            "descriptorOffset":descriptorOffset
+        }
+        return descriptorDictionnary
+class TriggerChannel(Channel):
+    """
+        Implementation of trigger control channel
+    """
+
+    triggerConfDict={}
+    
+    def getTriggerConfig(self):
+        """
+            Returns the current trigger configuration as a dict with keys:
+                - "type": condition that will trigger the acquisition of waveforms [EDGE,
+                slew,GLIT,intv,runt,drop]
+                - "source": trigger source (str, {EX,EX/5,C1,C2}) 
+                - "hold_type": hold type (refer to page 131 of programing guide)
+                - "hold_value1": hold value1 (refer to page 131 of programing guide)
+                - "level": Level at which the trigger will be set (float)
+                - "slope": (str,{POS,NEG,WINDOW}) Triggers on rising, falling or Window.
+                - "mode": behavior of the trigger following a triggering event (str, {NORM, AUTO, SINGLE,STOP}) 
+                - "coupling":  (str,{AC,DC}) Coupling to the trigger channel
+            and updates the internal configuration status
+        """
+        self.triggerConfDict.update(self.getSetup())
+        self.triggerConfDict.update(self.getLevel())
+        self.triggerConfDict.update(self.getSlope())
+        self.triggerConfDict.update(self.getMode())
+        self.triggerConfDict.update(self.getCoupling())
+        return self.triggerConfDict
+
+    def getSetup(self):
+        """
+            Returns the current trigger setup as a dict with keys:
+                - "type": condition that will trigger the acquisition of waveforms [EDGE,
+                slew,GLIT,intv,runt,drop]
+                - "source": trigger source (str, {EX,EX/5,C1,C2}) 
+                - "hold_type": hold type (refer to page 131 of programing guide)
+                - "hold_value1": hold value1 (refer to page 131 of programing guide)
+        """
+        get_process=lambda v:{
+            "type":v.split(" ",1)[1].split(",")[0],
+            "source":v.split(" ",1)[1].split(",")[2],
+            "hold_type":v.split(" ",1)[1].split(",")[4],
+            "hold_value1":v.split(" ",1)[1].split(",")[6][:-1],
+            }
+        triggerSetupDict=get_process(self.ask("TRSE?"))
+        return triggerSetupDict
+    def getLevel(self):
```

Comment:
One empty line between class definitions, please.
```suggestion

    def getLevel(self):
```